src/authorizations/actions/thunks.ts

```diff
@@ -166,3 +167,18 @@ export const deleteAuthorization = (id: string, name: string = '') => async (
     dispatch(notify(authorizationDeleteFailed(name)))
   }
 }
+
+export const getResources = () => async dispatch => {
+  try {
+    dispatch(setAllResources(RemoteDataState.Loading))
+    const resp = await api.getResources({headers: {}})
+
+    if (resp.status !== 200) {
+      throw new Error(resp.data.message)
+    }
+    const resources = resp.data
+    dispatch(setAllResources(RemoteDataState.Done, resources))
+  } catch (e) {
+    console.error(e)
+  }
```

Comment:
If we're not going to gracefully handle the error here, I think we should let it fail by removing this `catch` block and have the UI / component layer handle this with a notification

Reply:
Good idea